Dockerfile

```diff
@@ -45,33 +46,41 @@ RUN set -xe; \
     rm --recursive --force /var/lib/apt/lists/*;
 
 # Copy run scripts
-COPY scripts/docker/thunder-php-install /usr/local/bin/
-COPY scripts/docker/thunder-php-test /usr/local/bin/
-COPY scripts/docker/thunder-php-run /usr/local/bin/
+COPY scripts/docker/standard-php-install /usr/local/bin/
+COPY scripts/docker/standard-php-test /usr/local/bin/
+COPY scripts/docker/standard-php-run /usr/local/bin/
 
 # Set executable
 RUN set -xe; \
     \
-    chmod +x /usr/local/bin/thunder-php-install; \
+    chmod +x /usr/local/bin/standard-php-install; \
     \
-    chmod +x /usr/local/bin/thunder-php-test; \
+    chmod +x /usr/local/bin/standard-php-test; \
     \
-    chmod +x /usr/local/bin/thunder-php-run;
+    chmod +x /usr/local/bin/standard-php-run;
 
 # Copy pre-build Thunder project to container
 COPY --chown=thunder:thunder www /home/thunder/www
 
+# Build codebase.
+RUN set -xe; \
+    \
+    su - thunder --command="cd /home/thunder/www; COMPOSER_MEMORY_LIMIT=-1 composer install --no-dev"; \
+    \
+    su - thunder --command="cd /home/thunder/www; COMPOSER_MEMORY_LIMIT=-1 COMPOSER_ROOT_VERSION=${COMPOSER_ROOT_VERSION} composer require drush/drush thunder/thunder_performance_measurement thunder/testsite_builder drupal/media_entity_generic"; \
+    \
+    echo -e "\nexport PATH=\"\$PATH:/home/thunder/www/vendor/bin\"\n" >> /home/thunder/.profile; \
+    \
+    su - thunder --command="composer clear-cache";
+
+
 # Install Elastic APM
 RUN set -xe; \
     \
     su - thunder --command="cd /home/thunder/www/docroot/core; yarn add elastic-apm-node --dev"; \
     \
     su - thunder --command="yarn cache clean"; \
     \
-    su - thunder --command="composer global require drush/drush"; \
-    \
-    echo -e "\nexport PATH=\"\$PATH:/home/thunder/.composer/vendor/bin\"\n" >> /home/thunder/.profile; \
-    \
     su - thunder --command="composer clear-cache";
```

Comment:
Since you have moved usage of composer to another `RUN`, you can remove clear cache here.

Reply:
i removed the clear cache in the other run and kept this one